frontend/src/app/pages/exercises/exercise/shared/editor-panel/image-template-form/image-template-form.component.html

```diff
@@ -0,0 +1,59 @@
+<ng-container *ngIf="values">
+    <div class="form-group mb-3">
+        <label class="form-label">Bildadresse:</label>
+        <input
+            #imageUrlInput="ngModel"
+            [(ngModel)]="values.url"
+            [appAutofocus]="true"
+            appImageExistsValidator
+            required
+            type="url"
+            class="form-control"
+            placeholder="https://example.com/image.png"
+        />
+        <app-display-validation
+            [ngModelInput]="imageUrlInput"
+        ></app-display-validation>
+    </div>
+    <div class="form-group mb-3">
+        <label class="form-label">Name:</label>
+        <input
+            #nameInput="ngModel"
+            [(ngModel)]="values.name"
+            required
+            type="text"
+            class="form-control"
+            placeholder="Bildname"
+        />
+        <app-display-validation
+            [ngModelInput]="nameInput"
+        ></app-display-validation>
+    </div>
+    <div class="form-group mb-3">
+        <label class="form-label">Höhe:</label>
+        <input
```

Comment:
Sorry, but I'd like to have the input rejected before proposing anything if the height is not an integer.

Reply:
Well, I don't really think the cost/benefit justifies this feature, but ok I'll do it...